gateway/metrics/exporter.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Alex Ellis 2017. All rights reserved.
```

Comment:
If this involved a large amount of copy/paste then this header should be fine - otherwise add an additional line for OpenFaaS Author(s).

Reply:
Thanks Alex. I've added OpenFaaS Author(s) to the copyright.